src/app/public/modules/flyout/flyout.component.spec.ts

```diff
@@ -316,61 +317,86 @@ describe('Flyout component', () => {
   );
 
   it('should resize when handle is dragged', fakeAsync(() => {
-      openFlyout({});
-      const flyoutElement = getFlyoutElement();
-      const handleElement = getFlyoutHandleElement();
+    openFlyout({});
+    fixture.detectChanges();
+    tick();
+    const spy = spyOn(SkyFlyoutComponent.prototype, 'onMouseMove').and.callThrough();
+    const flyoutElement = getFlyoutElement();
+    const handleElement = getFlyoutHandleElement();
 
-      expect(flyoutElement.style.width).toBe('500px');
+    expect(flyoutElement.style.width).toBe('500px');
 
-      let evt = document.createEvent('MouseEvents');
-      evt.initMouseEvent('mousedown', false, false, window, 0, 0, 0, 1000,
-        0, false, false, false, false, 0, undefined);
+    let evt = document.createEvent('MouseEvents');
+    evt.initMouseEvent('mousedown', false, false, window, 0, 0, 0, 1000,
+      0, false, false, false, false, 0, undefined);
 
-      handleElement.dispatchEvent(evt);
-      makeEvent('mousemove', { clientX: 1100 });
-      fixture.detectChanges();
-      tick();
-      expect(flyoutElement.style.width).toBe('400px');
-      makeEvent('mousemove', { clientX: 1000 });
-      fixture.detectChanges();
-      tick();
-      expect(flyoutElement.style.width).toBe('500px');
-      makeEvent('mouseup', {});
+    handleElement.dispatchEvent(evt);
+    makeEvent('mousemove', { clientX: 1100 });
+    fixture.detectChanges();
+    tick();
+    expect(flyoutElement.style.width).toBe('400px');
+    makeEvent('mousemove', { clientX: 1000 });
+    fixture.detectChanges();
+    tick();
+    expect(flyoutElement.style.width).toBe('500px');
+    makeEvent('mouseup', {});
+    expect(spy).toHaveBeenCalled();
+  }));
+
+  it('should not resize on mousemove unless the resize handle was clicked', fakeAsync(() => {
+    openFlyout({});
+    fixture.detectChanges();
+    tick();
+    const spy = spyOn(SkyFlyoutComponent.prototype, 'onMouseMove').and.callThrough();
+    const flyoutElement = getFlyoutElement();
+
+    expect(flyoutElement.style.width).toBe('500px');
```

Comment:
There's several checks in here for widths. Do you really care about that in this particular test?

Seems like all you care about is the spy not called.

Reply:
Might also want to check mouseup?

Reply:
I could add the mouseup but I also want to check width just as a sanity check that nothing resized.